integration-tests/switch-wallet.test.ts

```diff
@@ -36,14 +36,14 @@ describe('Switch wallet', () => {
     await createNewWallet();
   });
 
-  it('expect to filter mainnet documents from testnet documents', async () => {
-    await setNetwork('mainnet');
-    await createWalletData();
-    await assertWalletData();
-    await setNetwork('testnet');
-    const isEmpty = await isWalletEmpty();
-    expect(isEmpty).toBe(true);
-  });
+  // it('expect to filter mainnet documents from testnet documents', async () => {
```

Comment:
is there a plan for restoring this test?

Reply:
Yes, https://dock-team.atlassian.net/browse/DCKA-1708. I will fix those today

Reply:
I will push another PR to fix that